### PR TITLE
Adds in Sheep vore mob.

### PR DESCRIPTION
**Pull request by H0lySquirr3l**
image
image

What says on tin.

Currently Admin spawn only until later update.

### CODE REVIEW

**Comment by Very-Soft** [comment]
Notably, there should probably be an alternate sprite for it being sheared if we want it to have that functionality eventually.

**Comment by KasparoVy** [comment]
Aw wow, adorable sprite!

**Comment by H0lySquirr3l** [comment]
Those sprites will be worked on, once the later update is added.
.yes, Rana did a good job!

File: vorestation.dme [hunk 1]
```diff
@@ -1,4065 +1,4066 @@
...
-#include "maps\submaps\surface_submaps\mountains\mountains_areas.dm"
-#include "maps\submaps\surface_submaps\mountains\mountains_areas_vr.dm"
-#include "maps\submaps\surface_submaps\plains\plains.dm"
-#include "maps\submaps\surface_submaps\plains\plains_areas.dm"
-#include "maps\submaps\surface_submaps\wilderness\wilderness.dm"
-#include "maps\submaps\surface_submaps\wilderness\wilderness_areas.dm"
-#include "maps\tether\tether.dm"
-#include "maps\~map_system\maps.dm"
-// END_INCLUDE
+// DM Environment file for baystation12.dme.
```
Comment (Arokha) [inline vorestation.dme:1]: Your PR changes all the line endings in vorestation.dme to LF instead of the proper-for-this-repo CRLF.

**Review by Arokha — CHANGES_REQUESTED** [review]

File: vorestation.dme (hunk 1 above)
Comment (H0lySquirr3l) [reply vorestation.dme:1]: Ok how do I accept your changes? Or is this a review that has changes in it?

Comment (Arokha) [reply vorestation.dme:1]: It doesn't have changes in it, I can't do a suggestion that's 4000+ lines long. :P

Comment (H0lySquirr3l) [reply vorestation.dme:1]: Then how do I fix it? I legit just downloaded and compiled it off a fresh copy of the (at the time) latest version of the server?

Comment (H0lySquirr3l) [reply vorestation.dme:1]: I touched nothing else.

Comment (Arokha) [reply vorestation.dme:1]: Your editor touched something else. You'll have to use a program like vscode or notepad++ or something that can change line endings

**Pull request closed without merging**